packages/reporter/src/runnables/runnables.scss

```
@@ -263,4 +263,65 @@
       margin-right: 5px;
     }
   }
+
+  .runnable-loading {
+    .runnable-loading-animation {
+      display: flex;
+      margin: 3.5rem auto 1.5rem;
+      padding: 0 5px;
+      width: 160px;
+
+      div {
+        animation: scaling 1.65s linear infinite;
```

```suggestion
        animation: scaling 1.65s ease-in-out infinite;
```

Smooths out the animation a little.

---

I had this originally but as I tried to mimic the spec I changed this to `linear`. However, the final version definitely does look better with it.